### PR TITLE
Edit/post

### PR DESCRIPTION
**Pull request by krismutalapova**
Changes for classes:
PostController.java    
PostRepository.java
PostService.java

### CODE REVIEW

**Review by MelanyCristy — APPROVED** [review]
I went through it and i couldn’t see if something was wrong.

**Comment by parpinelopi** [comment]
It looks alright.

**Review by parpinelopi — APPROVED** [review]
It looks alright.

File: src/main/java/se/kth/sda6/skeleton/posts/PostService.java [hunk 1]
```diff
@@ -2,33 +2,39 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 /*
     @TODO Autowire the PostRepository and use it to implement all the service methods.
  */
+@Service
 public class PostService {
+    @Autowired
+    private PostRepository repository;
```
Comment (SaraGarabedian) [inline src/main/java/se/kth/sda6/skeleton/posts/PostService.java:14]: repository would be more explicit if it was called postRepository (such as in CommentService, the name is commentRepository)

**Review by SaraGarabedian — COMMENTED** [review]
Seems good

**Pull request merged**